src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs

```diff
@@ -62,15 +62,21 @@ public async Task<ReviewModel> GetMasterReviewForPackageAsync(string language, s
             return reviews.FirstOrDefault();
         }
 
-        public async Task<IEnumerable<ReviewModel>> GetReviewsAsync(string language, string packageName, bool isAutomatic)
+        public async Task<IEnumerable<ReviewModel>> GetReviewsAsync(string language, string packageName = "", bool? isAutomatic = null)
         {
             var queryStringAutomaticFilter = "SELECT * FROM Reviews r WHERE " +
```

Comment:
What's the difference between these two queries. It's would be very hard to maintain them

Reply:
Fixed. I have reverted to single query that works for all scenarios.